test/main/modules.js

```diff
@@ -63,6 +90,31 @@ QUnit.test( "runs after final test", function( assert ) {
 	assert.expect( 1 );
 } );
 
+var ranAfterHook;
+
+QUnit.module( "after (skip)", {
+	after: function( assert ) {
+		ranAfterHook = true;
+		assert.ok( true, "after hook ran" );
+	}
+} );
+
+QUnit.test( "does not run after initial tests", function( assert ) {
+	assert.expect( 0 );
+	ranAfterHook = false;
+} );
+
+QUnit.skip( "skipped, but should still run after hook", function( assert ) {
+	assert.expect( 1 );
```

Comment:
this should not be necessary. Some throwing expression here, like `throw 'foo'` here would help better